compose-oneliner.sh

```diff
@@ -217,6 +217,17 @@ if ! ansible-playbook --become --become-user=root ansible/main.yml -vv; then
     exit 1
 fi
 
+
+function run_xhost {
+    if [ -z ${SUDO_USER} ]; then 
+        echo "SUDO_USER is blank, cannot retrive this parameter, mayabe running sudo with other profile (-)."
+    else 
+        echo "SUDO_USER is set to '$SUDO_USER', running xhost + with '$SUDO_USER'"
+        su - ${SUDO_USER} -c "xhost +"
```

Comment:
make it 'xhost +SI:localuser:root' insted of 'xhost +'

Reply:
changed.